info/src/main/java/com/popup/info/InfoCrawlerApplication.java

```diff
@@ -0,0 +1,23 @@
+package com.popup.info;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.data.mongodb.config.EnableMongoAuditing;
+
+import com.popup.info.service.CrawlerController;
+
+import lombok.AllArgsConstructor;
+
+@EnableMongoAuditing
+@AllArgsConstructor
+@SpringBootApplication
+public class InfoCrawlerApplication{
+
+	public static void main(String[] args) {
+		ConfigurableApplicationContext context = SpringApplication.run(InfoCrawlerApplication.class, args);
+		CrawlerController crawlerController = context.getBean(CrawlerController.class);
+		crawlerController.startCrawling();
```

Comment:
```suggestion
		SpringApplication.run(InfoCrawlerApplication.class, args);
```
스프링 컨텍스트를 직접 만들어서 빈을 수동으로 가져오는 방법보다 이미 주입된 빈을 사용하도록 `@PostConstruct`을 써서 구현해보는 건 어떨까요?